pyodbc.cpython-312-x86_64-linux-gnu.so,sha256=ChVrY5JiG7pJqoUdWsSdyEBpivF5-0Gb5SipCXXIRMM,997585
pyodbc.pyi,sha256=rx05N8EYqujKI3FWtMikUjqMPTv4VQg0ChXny9DaisE,32326
pyodbc-5.0.0.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
pyodbc-5.0.0.dist-info/WHEEL,sha256=4ZiCdXIWMxJyEClivrQv1QAHZpQh8kVYU92_ZAVwaok,152
pyodbc-5.0.0.dist-info/RECORD,,
pyodbc-5.0.0.dist-info/METADATA,sha256=UCCQ_QLPO9Mamlo69HQstSc1QzMfiCYTSUJZIm2clXA,2717
pyodbc-5.0.0.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
